sources/api/schnauzer/src/helpers.rs

```diff
@@ -463,6 +475,74 @@ pub fn ecr_prefix(
     Ok(())
 }
 
+/// `join_array` is used to join an array of scalar strings into an array of
+/// quoted, delimited strings. The delimiter must be specified.
+///
+/// # Example
+///
+/// Consider an array of values: `[ "a", "b", "c" ]` stored in a setting such as
+/// `settings.somewhere.foo-list`. In our template we can write:
+/// `{{ join-array "," settings.somewhere.foo-list }}`
+///
+/// This will render `"a", "b", "c"`.
+pub fn join_array(
+    helper: &Helper<'_, '_>,
+    _: &Handlebars,
+    _: &Context,
+    renderctx: &mut RenderContext<'_, '_>,
+    out: &mut dyn Output,
+) -> Result<(), RenderError> {
+    trace!("Starting join-array helper");
+    let template_name = template_name(renderctx);
+    check_param_count(helper, template_name, 2)?;
+
+    // get the delimiter
+    let delimiter_param = get_param(helper, 0)?;
+    let delimiter = delimiter_param
+        .as_str()
+        .with_context(|| error::JoinStringsWrongType {
+            expected_type: "string",
+            value: delimiter_param.to_owned(),
+            template: template_name,
+        })?;
+
+    // get the array
+    let array_param = get_param(helper, 1)?;
+    let array = array_param
+        .as_array()
+        .with_context(|| error::JoinStringsWrongType {
+            expected_type: "array",
+            value: array_param.to_owned(),
+            template: template_name,
+        })?;
+
+    let mut result = String::new();
+    for (i, value) in array.iter().enumerate() {
+        if i > 0 {
+            result.push_str(delimiter);
+            result.push(' ');
```

Comment:
Hmm.  Should we force a space here, rather than the user specifying `", "` if they want a space?